dev-python/snakemake/snakemake-8.25.3.ebuild

```diff
@@ -71,4 +55,17 @@ RDEPEND="
 # 	dev-python/docutils \
 # 	dev-python/recommonmark \
 # 	dev-python/myst-parser
+
+EPYTEST_IGNORE=(
+	# __file__ attribute does not match test name ('python_3_7')
+	tests/test_conda_python_3_7_script/test_script.py
+	# ModuleNotFoundError: No module named 'snakemake_executor_plugin_cluster_generic'
+	tests/test_executor_test_suite.py
+	# ImportError: cannot import name 'snakemake' from 'snakemake.script'
+	tests/test_script_py/scripts/test_explicit_import.py
```

Comment:
Does setting `PYTHONPATH` (or similar) help for this one?

Reply:
If I understand [this](https://github.com/snakemake/snakemake/issues/2961#issuecomment-2258991074) comment correctly, it is not a matter of simply adding to `PYTHONPATH`:

> running `from snakemake.script import snakemake` outside of a Snakemake "context", since `snakemake.script.snakemake` (lower case) is simply a type hint that can be used by editor. At execution time, the `snakemake` object needs to be created by snakemake, which is done by the boilerplate/preamble

Reply:
That comment seems to suggest that passing some path to `epytest` would resolve the problem, have you tried this?